test/node-test/abort-controller.js

```diff
@@ -1,11 +1,82 @@
 'use strict'
 
-const { test } = require('tap')
+const { test } = require('node:test')
 const { AbortController: NPMAbortController } = require('abort-controller')
-const { Client, errors } = require('..')
+const { Client, errors } = require('../..')
 const { createServer } = require('http')
 const { createReadStream } = require('fs')
-const { wrapWithAsyncIterable } = require('./utils/async-iterators')
+const { wrapWithAsyncIterable } = require('../utils/async-iterators')
+const { tspl } = require('@matteo.collina/tspl')
+
+/**
+ * A port of tap's `t.type` that can be used with `node:assert`
+ * https://github.com/tapjs/tapjs/blob/511019b2ac0fa014370154c3a341a0e632f50b19/src/asserts/src/index.ts#L199
+ */
+function ttype (plan, obj, klass) {
```

Comment:
Where we should put helper function equivalent to tap's `t.type` ? I think it is necessary to at least extract it into a separate file.